### PR TITLE
General coding standard check

### LINKED ISSUE #346: Text file syntax reminder output format
image

This is the messages shown when I start the application with invalid data in all three data files.  I noticed that there is no new line to separate the error msg and the syntax for the next text file (for example: `menu.txt: Invalid format...` is immediately followed by `Format for Orders.txt:`. 

May I know is this behaviour intended? Would the message be clearer if there was a line to separate messages for different text files?

### PR DESCRIPTION
**Pull request by ziyi105**
Fixes #346

### CODE REVIEW

File: src/main/java/seedu/cafectrl/data/CurrentDate.java [hunk 1]
```diff
@@ -3,24 +3,26 @@
 import java.util.ArrayList;
 
 public class CurrentDate {
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
```
Comment (DextheChik3n) [inline src/main/java/seedu/cafectrl/data/CurrentDate.java:7]: perhaps you could name these variables with something more meaningful such as:
```suggestion
    private static final int MIN_ORDER_LIST_SIZE = 0;
    private static final int DAY_BASE_NUMBER = 0;
    private static final int DAY_OFFSET = 1;
    private static final int ORDER_LIST_SIZE_OFFSET = 1;
```

File: src/main/java/seedu/cafectrl/data/dish/Ingredient.java [hunk 2]
```diff
@@ -38,8 +38,16 @@ public String toString() {
...
+    /**
+     * Compare the name of this ingredient to the other ingredient
+     * @param obj the other ingredient to be compared with
+     * @return true if they have the same name, false otherwise
+     */
     @Override
     public boolean equals(Object obj) {
-        return this.toString().equals(obj.toString());
+        if (obj instanceof Ingredient) {
+            return this.name.equals(((Ingredient) obj).name);
```
Comment (DextheChik3n) [inline src/main/java/seedu/cafectrl/data/dish/Ingredient.java:49]: perhaps you want to create variable for `(Ingredient) obj).name` to simplify this statement

**Review by DextheChik3n — COMMENTED** [review]
Added some comments on some small suggested changes

**Review by DextheChik3n — APPROVED** [review]
Good job on ensuring the spacing between processes to ensure clarity in code and replacing magic numbers.

**Review by ShaniceTang — COMMENTED** [review]
Looks good! Thanks :)

**Pull request merged**